docs/index.bs

```diff
@@ -3154,7 +3150,33 @@ spec: storage; urlPrefix: https://storage.spec.whatwg.org/
                       1. If |client| is not null, |response|'s [=response/type=] is "`opaque`", and [=cross-origin resource policy check=] with |request|'s [=request/origin=], |client|, "", and |response|'s [=filtered response/internal response=] returns <b>blocked</b>, then return null.
                       1. Return |response|.
               1. Return null.
-      1. If |request| is a <a>non-subresource request</a>, |request| is a [=navigation request=], |registration|'s [=navigation preload enabled flag=] is set, |request|'s [=request/method=] is \`<code>GET</code>\`, |registration|'s [=active worker=]'s [=set of event types to handle=] [=set/contains=] <code>fetch</code>, and |registration|'s [=active worker=]'s [=all fetch listeners are empty flag=] is not set then:
+          1. Else if |source| is {{RouterSourceEnum/"race-network-and-fetch-handler"}}, and |request|'s [=request/method=] is \`<code>GET</code>\` then:
+
+              Note: In order to avoid duplicated fetches to |request|, the user agent may reuse |raceNetworkRequestResponse| as the result of {{FetchEvent}}'s [=FetchEvent/potential response=], or the fallback request.
+
+              1. Let |queue| be an empty [=queue=] of [=/response=].
+              1. Let |raceFetchController| be null.
+              1. Run the following substeps [=in parallel=], but [=abort when=] |controller|'s [=fetch controller/state=] is "<code>terminated</code>" or "<code>aborted</code>":
+                  1. Set |raceFetchController| to the result of [=Fetch|fetching=] |request|.
+                  1. To [=fetch/processResponse=] for |raceNetworkRequestResponse|, run these substeps:
+                      1. If |raceNetworkRequestResponse|'s [=response/type=] is "`error`", then:
+                          1. Terminate these substeps.
+                      1. If |raceNetworkRequestResponse|'s [=response/status=] is `200`, then:
+                          1. [=queue/Enqueue=] |raceNetworkRequestResponse| to |queue|.
+              1. [=If aborted=] and |raceFetchController| is not null, then:
```

Comment:
"If aborted" can only be used inside "abort when". But this is a sibling step to "abort when". Maybe indentation got messed up?